packages/clay-charts/src/Chart.js

```diff
@@ -8,10 +8,10 @@ import templates from './Chart.soy.js';
  * @extends Component
  * @extends ChartBase
  */
-class Chart extends Component {}
+class Chart extends DataComponent {}
 
 Object.assign(Chart.prototype, ChartBase);
-Chart.STATE = ChartBase.STATE;
+Chart.STATE = Object.assign({}, ChartBase.STATE, DataComponent.STATE);
```

Comment:
I think that we probably don't need to merge `DataComponent.STATE` here, since metal will pick it up from the class chain. Could you double check?

Reply:
Yes you're right, this is not needed. Thanks.